src/coreComponents/physicsSolvers/wavePropagation/AcousticElasticWaveEquationSEM.hpp

```diff
@@ -175,8 +175,8 @@ class AcousticElasticWaveEquationSEM : public CoupledWaveSolver< AcousticWaveEqu
   virtual void initializePostInitialConditionsPreSubGroups() override;
 
   SortedArray< localIndex > m_interfaceNodesSet;
-  arrayView1d< string const > m_acousRegions;
-  arrayView1d< string const > m_elasRegions;
+  string_array const * m_acousRegions;
+  string_array const * m_elasRegions;
```

Comment:
do we need raw pointers here? if not we might prefer a weak pointer type or a reference wrapper?